libraries/chain/include/eosio/chain/block_header_state.hpp

```diff
@@ -25,6 +26,7 @@ struct block_header_state {
     public_key_type                   block_signing_key;
     vector<uint8_t>                   confirm_count;
     vector<header_confirmation>       confirmations;
+    std::shared_future<public_key_type> block_signing_key_future;
```

Comment:
This doesn't seem to be used?  was there some files left out of this PR (despite it operating fine)?  Or is this future expansion

Reply:
I was experimenting with block signing future. I'll remove this.